src/ui/layout/constraints.md

```diff
@@ -1973,26 +2022,26 @@ If you want the `Scaffold`'s child to be exactly the same size
 as the `Scaffold` itself, you can wrap its child with
 `SizedBox.expand`.
 
-{{site.alert.note}}
-  When a widget tells its child that it must be of
-  a certain size, we say the widget supplies _tight_
-  constraints to its child.
-{{site.alert.end}}
-
-
-## Tight vs. loose constraints
+## Tight vs loose constraints
 
 It’s very common to hear that some constraint is
-"tight" or "loose", so it’s worth knowing what that means.
+"tight" or "loose", so what does that mean?
 
 A _tight_ constraint offers a single possibility,
 an exact size. In other words, a tight constraint
 has its maximum width equal to its minimum width;
 and has its maximum height equal to its minimum height.
 
+An example of this is the `App` widget,
+which is contained by the [`RenderView`][] class:
+the box used by the child returned by the
+application's [`build`][] function is given a constraint
+that forces it to exactly fill the application's content area
+(typically, the entire screen).
```

Comment:
maybe this is where that note earlier about the root tight constraints should go?